examples/computer_vision_techniques/mask_disk.py

```diff
@@ -6,35 +6,33 @@
 How to mask out all emission from the solar disk.
 """
 import matplotlib.pyplot as plt
-import numpy as np
-import numpy.ma as ma
 
 import sunpy.map
 from sunpy.data.sample import AIA_171_IMAGE
-from sunpy.map.maputils import all_coordinates_from_map
+from sunpy.map.maputils import all_coordinates_from_map, coordinate_is_on_solar_disk
 
 ###############################################################################
 # We start with the sample data
 aia = sunpy.map.Map(AIA_171_IMAGE)
 
 ###############################################################################
 # A utility function gives us access to the helioprojective coordinate of each
-# pixels. We can use that to create a new array which
-# contains the normalized radial position for each pixel.
+# pixels. We can use that to create a new array of all the coordinates
+# that are on the solar disk.
 hpc_coords = all_coordinates_from_map(aia)
-r = np.sqrt(hpc_coords.Tx ** 2 + hpc_coords.Ty ** 2) / aia.rsun_obs
 
 ###############################################################################
-# With this information, we create a mask where all values which are less then
-# the solar radius are masked. We also make a slight change to the colormap
+# Now, we can create a mask from the coordinates by using another utility
+# function that gives us a mask that contains all the coordinates that are on
+# the solar disk(marked by True). We also make a slight change to the colormap
 # so that masked values are shown as black instead of the default white.
```

Comment:
```suggestion
# function that gives us a mask that has `True` for those coordinates that are
# on the solar disk. We also make a slight change to the colormap so that
# masked values are shown as black instead of the default white.
```